### PR TITLE
Add app.drey.EarTag

### PR DESCRIPTION
**Pull request by knuxify**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub.
- [X] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge). (https://gitlab.gnome.org/sophie-h/app.drey/-/issues/5)
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(Flatpak manifest copied from upstream.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by knuxify** [comment]
bot, build app.drey.EarTag

File: app.drey.EarTag.json [hunk 1]
```diff
@@ -0,0 +1,57 @@
+{
+    "app-id" : "app.drey.EarTag",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "42",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "eartag",
+    "finish-args" : [
+        "--filesystem=xdg-pictures:ro",
+        "--filesystem=xdg-music:rw",
+        "--filesystem=xdg-download:rw",
```
Comment (barthalion) [inline app.drey.EarTag.json:10]: Are you sure your app does not use a portal already?

Comment (knuxify) [reply app.drey.EarTag.json:10]: If I understand the docs correctly, it *should* be using a portal becuase I'm using GtkFileChooserNative, but for some reason it wasn't doing so on my system when I initially tested it.

**Comment by knuxify** [comment]
bot, build app.drey.EarTag

**Pull request closed without merging**